assets/queries/terraform/kubernetes_pod/rbac_roles_with_read_secrets_permissions/query.rego

```diff
@@ -0,0 +1,31 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource[resourceType]
```

Comment:
According to the k8s version of this query, only two resource types needs to be checked: Role and ClusterRole
```suggestion
	resourceTypes := ["kubernetes_role", "kubernetes_cluster_role"]
	resource := input.document[i].resource[resourceTypes[t]][name]
```